upi/src/main/res/layout/upi_app.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="utf-8"?><!--
+  ~ Copyright (c) 2024 Adyen N.V.
+  ~
+  ~ This file is open source and available under the MIT license. See the LICENSE file for more info.
+  ~
+  ~ Created by ararat on 12/3/2024.
+  -->
+
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/layout_upi_app"
+    style="@style/AdyenCheckout.UPI.RecyclerListItem"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content">
+
+    <com.adyen.checkout.ui.core.internal.ui.view.RoundCornerImageView
+        android:id="@+id/imageView_upi_logo"
+        style="@style/AdyenCheckout.UPI.Intent.Icon"
+        app:layout_constraintBottom_toTopOf="@+id/divider_upi_app"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <androidx.appcompat.widget.AppCompatTextView
+        android:id="@+id/textView_upi_app_name"
+        style="@style/AdyenCheckout.UPI.Intent.AppIdTextView"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        app:layout_constraintBottom_toTopOf="@+id/divider_upi_app"
+        app:layout_constraintEnd_toStartOf="@+id/radioButton_upi_app"
+        app:layout_constraintHorizontal_bias="0"
+        app:layout_constraintStart_toEndOf="@+id/imageView_upi_logo"
+        app:layout_constraintTop_toTopOf="parent"
+        tools:text="GooglePay" />
+
+    <com.google.android.material.radiobutton.MaterialRadioButton
+        android:id="@+id/radioButton_upi_app"
+        style="@style/AdyenCheckout.UPI.Intent.RadioButton"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:clickable="false"
+        app:layout_constraintBottom_toTopOf="@+id/divider_upi_app"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <View
```

Comment:
I did not get this part. Could you explain what is not centered?

Reply:
![image](https://github.com/Adyen/adyen-android/assets/17701279/8f623b6f-1c30-472b-90e3-c3245a4da960)

There is quite some more space on top

Reply:
I think that is from the font. Since it is the same in payment methods list.
<img width="238" alt="Screenshot 2024-06-12 at 16 35 05" src="https://github.com/Adyen/adyen-android/assets/8832594/93d375a9-5643-4d36-8b31-ca6843b61198">

Reply:
I updated the designs